### ISSUE TITLE
Accessibility - About this App Screen - Missing Heading

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#24]
Why are there no headings on the ‘About This App’ screen? They were on the iOS application and I thought content was the same for this screen.

**Comment by aquilent-dgemoets** [comment]
The headings are there, but this appears to be how TalkBack currently reads HTML headings in standard Android WebViews.  We use the same HTML as the iOS version and put it in the standard Android WebView UI element.

This also applies to #27.

**Comment by jfc3-dol** [comment]
I reviewed the ILAB 'Sweat and Toil' application on OPA's 'Galaxy Tab S' and an ILAB employees 'LG Tribute 2' and both did not read any headings. 

From my understanding two of the 'More information' screens (About the App and Methodology) where HTML pages and had the headings in them. 

I thought the OSHA 'Heat Index' application had similar screens and the headings with their proper level were read for HTML screens before. The other Android screens that had what equated to headings were read too.

**Comment by aquilent-dgemoets** [comment]
The (unpublished) Android version of the OSHA 'Heat Index' application uses the same WebViews that Sweat & Toil does.  I'd be surprised if TalkBack reads HTML headings as such.  Could you confirm?

**Comment by jfc3-dol** [comment]
Okay, I will try looking at the OSHA Heat Index application again once I'm done reviewing this one.

**Comment by jfc3-dol** [comment]
The "About this App" screen is now reading proper headings so I'm closing this issue.
